### ISSUE TITLE
Sixsense: crash to desktop on viewing list of queue for community map

### ISSUE THREAD

**Issue by LPChip** [ValveSoftware/portal2#230]
I own a sixense and for quite some time now, whenever I go to Portal 2->Community Maps->Play single player maps / play co-op maps, the game crashes as soon as it loads the first map in my queue.
![portal2 2014-08-05 17-48-00-11](https://cloud.githubusercontent.com/assets/8420917/3882056/bca5865e-2193-11e4-994d-04570c42daeb.jpg)

If I run the game with -nosixense option, everything works. I have reformatted windows 7 and it still happens. Also reinstalled steam and portal2, doesn't fix the issue.

I've submitted a ticket to steam support under 1729-UIFV-4537 but as usual they have no clue what this is and trying me to do things not related to the problem.

In a desperate search, I ended up here, and read about removing the client.dll and server.dll files from the portal2_sixense/bin folder, and that fixes my issue, but also disables my sixense controller. I'd like to be able to use my sixense controller with community maps again. If I have to with an earlier build/update.

OS: Windows 7 Ultimate x64
Version info:
Protocol version 2001
Exe version 2.0.0.1 (portal2)
Exe build: 16:00:43 Jul  9 2014 (5723) (620)

client_sixense.dll and server_sixense.dll are dated from 17-06-2014 / 19:46 GMT+2
client_sixesnse.dll is 11,333,632 bytes big
server_sixense.dll is 9,024,512 bytes big.

**Comment by LPChip** [comment]
HUGE UPDATE:
I've been playing with some old backups I have of portal2 by replacing single files or the entire bin folder of my portal2_sixense folder to see if I was able to fix my issue. I get different results with different files, but unfortunately they all crash. If I remove the client.dll or server.dll they usually work or have different symptoms, such as not having the sixense options and controller support.

That made me wonder. I restored my settings to the latest build and fired up the game to see if going to the options->Motion Controller options would crash the game, and it doesn't. Strangely enough, the game did not crash on displaying my queue either, and I can actually use the motion controller in-game in a workshop map.

I restarted the game, went to my queue, and bam. crash.

I restarted the game, went to my sixense options, heard the distinct noise (it gives a different beep if I go there), and then back to the queue, and there we go, it works again, and I can play the game.

So long story short, somehow an initialization routine is missing causing the game to crash. I'll just visit my options screen each time I want to play portal2 until this is solved. Hope this can be solved soon, as it seem to be an easy fix, just add one line of code to initialize the controller.

**Comment by LPChip** [comment]
Post made in the Portal2 forums: http://forums.steampowered.com/forums/showthread.php?p=35685744#post35685744

**Comment by vrad-exe** [comment]
This is still an issue

**Comment by LPChip** [comment]
Yup, unfortunately it is. I've stopped playing Portal2 because of this issue. Love to play it again but if this is not fixed, its just not fun. There are better games out there to use with a Sixense motion controller. Portal 2 used to be the absolute best, its now the absolute worst.

**Comment by vrad-exe** [comment]
I found something which said that the Sixense DLC has a "bug" if `maps/puzzlemaker/preview.bsp` exists. I haven't tried anything and have no idea what this "bug" is, but maybe it was referring to this?
